solr/core/src/test/org/apache/solr/update/processor/TolerantUpdateProcessorTest.java

```diff
@@ -57,12 +61,20 @@ public class TolerantUpdateProcessorTest extends UpdateProcessorTestBase {
   @BeforeClass
   public static void beforeClass() throws Exception {
     initCore("solrconfig-update-processor-chains.xml", "schema12.xml");
+    server = new EmbeddedSolrServer(h.getCoreContainer(), h.getCore().getName());
   }
 
   @AfterClass
-  public static void tearDownClass() {
+  public static void afterClass() {
     docs = null;
-    badIds = null;
+    if (server != null) {
```

Comment:
no need.  `server` is an `EmbeddedSolrServer` constructed with a `CoreContainer` originating from `TestHarness.  It's a thin veneer on something whose lifecycle is already managed (i.e. lifecycle like close).

It'd be nice if TestHarness had a `client` field (I'd call it that) so that tests cold use the familiar SolrClient.  Granted I want TestHarness to go away entirely.